README.md

```diff
@@ -0,0 +1,101 @@
+# F-Droid Repomaker's Flatpak
+
+This project contains the source files to create the
+[FlatPak](https://flatpak.org/) for
+[F-Droid Repomaker](https://f-droid.org/repomaker/).
+
+With Repomaker, you can easily create your own
+[F-Droid](https://f-droid.org) repo without needing any special
+knowledge to do so.
+
```

Comment:
A lot of what's in this README is a duplicate of what's in the flatpak documentation. It's better to remove them from here and just point to the documentation, if the documentation lacks some info, it should be reported so we can have something easy to read & understand. 
It's just to avoid having an old documentation with commands that don't work anymore.